src/components/PersonLink/PersonLink.tsx

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+import { NavLink } from 'react-router-dom';
+import classNames from 'classnames';
+import { Person } from '../../types';
+
+interface Props {
+  person: Person;
+}
+
+export const PersonLink: React.FC<Props> = ({ person }) => {
+  const { slug, name, sex } = person;
+
+  const FEMALE = 'f';
```

Comment:
you create this variable on each iteration.
I think you can move it outside of your component to avoid this behavior.